tee-worker/core/parentchain/indirect-calls-executor/Cargo.toml

```diff
@@ -61,8 +53,8 @@ itp-test = { path = "../../../core-primitives/test" }
 itp-top-pool-author = { path = "../../../core-primitives/top-pool-author", features = ["mocks"] }
 itc-parentchain-test = { path = "../../../core/parentchain/test" }
 
-# litentry
-hex = "0.4.2"
+# scs/integritee
+substrate-api-client = { git = "https://github.com/scs/substrate-api-client.git", branch = "polkadot-v0.9.39", default-features = false }
```

Comment:
@kziemianek did you manually switch this line or by the tool?

Reply:
Manually.

This module is referenced only in tests.